services/worker/src/worker/job_runners/config/size.py

```diff
@@ -58,6 +64,13 @@ def compute_config_size_response(dataset: str, config: str) -> ConfigSizeRespons
                 "num_bytes_memory": split_info["num_bytes"] if "num_bytes" in split_info else 0,
                 "num_rows": split_info["num_examples"] if "num_examples" in split_info else 0,
```

Comment:
[I know it's unrelated to this PR, but it's the occasion to look at this part of the code]

we should switch to None if we don't have the info, instead of imagining a false value (0), right?

Reply:
`datasets` always return `num_examples` so we're good, it was surely for mypy

Reply:
aaaaah ok